cmd/kubeops/main.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/kubeapps/kubeapps/cmd/kubeops/internal/handler"
+	"github.com/kubeapps/kubeapps/pkg/agent"
+	"github.com/kubeapps/kubeapps/pkg/auth"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/pflag"
+	"github.com/urfave/negroni"
+	"k8s.io/helm/pkg/helm/environment"
+)
+
+const driverEnvVar = "HELM_DRIVER"
+const defaultHelmDriver agent.DriverType = agent.Secret
+
+var (
+	settings         environment.EnvSettings
+	chartsvcURL      string
+	helmDriverArg    string
+	userAgentComment string
+	listLimit        int
+	timeout          int64
+)
+
+func init() {
+	settings.AddFlags(pflag.CommandLine) // necessary???
+	pflag.StringVar(&chartsvcURL, "chartsvc-url", "https://kubeapps-internal-chartsvc:8080", "URL to the internal chartsvc")
+	pflag.StringVar(&helmDriverArg, "helm-driver", "", "which Helm driver type to use")
+	pflag.IntVar(&listLimit, "list-max", 256, "maximum number of releases to fetch")
+	pflag.StringVar(&userAgentComment, "user-agent-comment", "", "UserAgent comment used during outbound requests") // necessary???
+	// Default timeout from https://github.com/helm/helm/blob/b0b0accdfc84e154b3d48ec334cd5b4f9b345667/cmd/helm/install.go#L216
+	pflag.Int64Var(&timeout, "timeout", 300, "Timeout to perform release operations (install, upgrade, rollback, delete)")
+}
+
+func main() {
+	pflag.Parse()
+	settings.Init(pflag.CommandLine)
+
+	options := agent.Options{
+		ListLimit: listLimit,
+		Timeout:   timeout,
+	}
+
+	// Will panic below if an invalid driver type is provided.
+	driverType := defaultHelmDriver
+	if helmDriverArg != "" {
+		driverType = agent.ParseDriverType(helmDriverArg)
+	} else {
+		// CLI argument was not provided; check environment variable.
+		helmDriverEnv := os.Getenv(driverEnvVar)
+		if helmDriverEnv != "" {
+			driverType = agent.ParseDriverType(helmDriverEnv)
+		}
+	}
```

Comment:
We did that because [Helm did](https://github.com/helm/helm/blob/f1dc84773f9a34fe59a504fdb32428ce1d56a2e8/cmd/helm/helm.go#L70). I'll be happy to remove the env var if you prefer just the flag.